src/screens/Favorite.js

```diff
@@ -1,11 +1,15 @@
 import StorageManager from '../services/StorageManager';
+import { TilesGrid } from '../components/Tiles/TilesGrid';
 import { render } from '../shared/dom';
 
 const template = `
-    <div></div>
+    <div>
+        <template id="grid-template"></template>
+    </div>
 `;
 
 export const FavoriteScreen = ({ renderOn }) => {
-    render({ html: template, on: renderOn });
     const favorites = StorageManager.getItem('favorites') || [];
+    render({ html: template, on: renderOn });
+    TilesGrid({ renderOn: '#grid-template', favorites });
```

Comment:
It doesn't work because you don't pass `movies` parameter into TilesGrid. Instead you pass `favorites` parameter, making `movies` undefined. You should change that line to `movies: favorites`.